charts/trino/values.yaml

```diff
@@ -90,8 +90,18 @@ server:
 accessControl: {}
 # accessControl -- [System access
 # control](https://trino.io/docs/current/security/built-in-system-access-control.html)
-# configuration.
+# configuration.<br>
+# `type: configmap | custom`<br>
```

Comment:
Remove this line, or change this into a proper English sentence, like so:
> Set the `type` property to either `configmap` and provide the rule file contents in `rules`, or `custom` and provide configuration properties in `properties`.

The examples should be sufficient in explaining how to set the type.